.github/workflows/build-test-lint.yml

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
I'm not familiar with actions, but should we be setting the CI env var to true? It's a matter of course for all my CI scripts.

Reply:
Good call, but I believe GH Actions already sets this as default, so no need to specify: https://docs.github.com/en/actions/reference/environment-variables

Good eyes though 👀